### DISCUSSION TITLE
Frequent HC Offline Problem

### DISCUSSION THREAD

**Discussion by qiangjiadong** [mozilla/hubs#3918] [Hubs Cloud Questions]
We are using AWS Personal Hubs Cloud. https://xartdistrict.com/

It seems every day or every 2 days, as long as I have time to check the link, it always goes offline around 4 AM NYC time (17 PM Beijing time).

Today I imported legacy .spoke and published a new scene. When I created a room and replaced with this new scene, suddenly my Hubs Cloud went offline.

It seems very strange since last Nov. as we also contacted AWS staff they were saying server is working fine without any problem...

Help please!

Thks!

**Comment by qiangjiadong** [comment]
https://hubs.mozilla.com/KPLE2zS/ The scene used in this room was imported into my Hubs Cloud that caused instant Off Line. I guess it is because of short of memory from our server...

If there is a way to see server logs or if that means maybe we need to update to Enterprise version for bigger RAM etc...

Thks!

**Comment by lihong-xmuseum** [comment]
At first, we found that the machine was offline after we deployed it. Later, we found that the machine was always offline at around 4:00 AM New York time (around 17:00 PM Beijing time) no matter whether it was deployed or not, which caused us a lot of trouble. Could you please tell us the reason or check the location of the server log?
